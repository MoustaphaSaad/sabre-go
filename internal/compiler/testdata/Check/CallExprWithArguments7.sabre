package main

func foo() (int, int) {
	return 1, 2
}

func foo2() (int, int) {
	return foo()
}

func bar(a, b, c int) {
}

func main() {
	bar(foo2(), 1)
}
