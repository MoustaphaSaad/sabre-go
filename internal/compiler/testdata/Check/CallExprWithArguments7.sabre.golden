>> 		bar(foo2(), 1)
>> 		^^^^^^^^^^^^^^ 
Error[internal/compiler/testdata/Check/CallExprWithArguments7.sabre:15:2]: expected 3 arguments, but found 2
>> 		bar(foo2(), 1)
>> 		^^^^^^^^^^^^^^ 
Note[internal/compiler/testdata/Check/CallExprWithArguments7.sabre:15:2]: have ((int,int),int), want (int,int,int)
